src/pages/VenuePage/VenuePage.tsx

```diff
@@ -230,7 +231,11 @@ const VenuePage: React.FC = () => {
     history.push(`/account/profile?venueId=${venueId}`);
   }
 
-  return <TemplateWrapper venue={venue} />;
+  return (
+    <CustomSoundsProvider>
+      <TemplateWrapper venue={venue} />
+    </CustomSoundsProvider>
```

Comment:
Having this here will only allow us to use custom sounds for templates in the client-side. Given right now we barely make use of the feature, and the only place we do is on `Rooms` on the `Map` this may not be super relevant.

But I can imagine situations when we would also want to be able to access these sounds from the admin panel (eg. to preview what they sound like, etc). In which case, this wouldn't make sense to have here.

As mentioned in another comment (and as followed originally in #1279), I think keeping this in `src/index.tsx` may be a better location.

Note that in #1279 the 'nesting order' is changed so that `CustomSoundsProvider` is a child of `AuthIsLoaded`, which will prevent the errors we are seeing.

---

Note also that for an unknown reason, #1279 seemed to break sounds in some cases. I expect it may be due to the loading/etc stuff you included in this PR or similar, but we didn't dig deeply into it at the time.

This is summarised on https://github.com/sparkletown/internal-sparkle-issues/issues/591:

> A small fix was made in https://github.com/sparkletown/sparkle/pull/1279 to ensure that we don't try and fetch the sounds till after `AuthIsLoaded` has finished running, as the security rules we have configured require a user to be authenticated before the can fetch the sounds data. This was merged directly into `env/kotr`.
> 
> Seemingly this caused issues with the sounds working within the `env/kotr` environment ([slack thread](https://doesitsparkle.slack.com/archives/C01ER4M61TL/p1616080529025000)), where they would work when you clicked on 'enter', but wouldn't work when you initially click on the room from the map ([slack ref](https://doesitsparkle.slack.com/archives/C01ER4M61TL/p1616083018026800?thread_ts=1616080529.025000&cid=C01ER4M61TL))
> 
> While he couldn't figure out the 'why' of it, @mike-lvov fixed the sound issues in https://github.com/sparkletown/sparkle/pull/1286 by essentially reverting the changes made by https://github.com/sparkletown/sparkle/pull/1279, which was deployed directly to the `env/kotr` environment.
> 
> This seemed to fix the issue in the `env/kotr` environment ([slack thread](https://doesitsparkle.slack.com/archives/C01K319QN5T/p1616086739085500)), but we still aren't really sure why those changes 'caused issues' in the first place.
>

Reply:
working on it. try to understand why it happens